maas-schemas/schemas/core/components/personalDataValidation.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$id": "http://maasglobal.com/core/components/personalDataValidation.json",
+  "description": "MaaS personal data validation rules",
+  "type": "object",
+  "properties": {
+    "type": {
+      "enum": ["value", "enum", "length", "regex", "date", "dateDurationTillNow"]
```

Comment:
What is a `dateDurationTillNow`?

Reply:
it's how long it has been since the date from personal data item
use case: age > 21, can be deferred from DOB

Reply:
so `dateDurationTillNow >= 7665 days` for example.

Reply:
It should be `days` not `date` :)

Reply:
@hieuunguyeen is this in days or is this `Date.now() - date`?

Reply:
May I also have array?

Reply:
@stodan it can be in any time unit, that is what the config object needs to define

Reply:
ok I searched and the answer is that Till also mean Until but doesn't mean Until (English strikes again)
I'll amend